test/test_helper.rb

```diff
@@ -7,16 +7,7 @@
 
 require "roger_sassc"
 
-RogerSassc.append_path "test/fixtures"
-
-module FixtureHelper
-  FIXTURE_ROOT = File.expand_path(File.join(File.dirname(__FILE__), "fixtures"))
+require File.dirname(__FILE__) + '/helpers/fixture_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.